frontend/public/components/utils/owners-resource-link.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { K8sResourceKind, referenceForOwnerRef } from '../../module/k8s';
+import { ResourceLink } from './index';
+
+export const OwnersResourceLink: React.FC<OwnersResourceLinkProps> = ({resource}) => {
```

Comment:
naming nit: I'd probably just call this `OwnerReferences` (and change the filename as well)